tests/txn.spec.ts

```diff
@@ -66,6 +66,51 @@ describe("txn", () => {
             const p = txn.query('{ me(func: eq(name, "Alice")) { name }}');
             await expect(p).rejects.toBe(dgraph.ERR_FINISHED);
         });
+
+        it("should succeed without increasing startTs (readOnly=true)", async () => {
+            const res = await client.newTxn().queryWithVars(
```

Comment:
using `txn.query()` and inlining `$a` will make this test shorter.

Reply:
This is easy, I can change both of these.